### ISSUE TITLE
Save file is folder 

### ISSUE THREAD

**Issue by Crosshairjy** [metehankutlu/decky-save-manager#5]
Currently I'm playing a game that the whole save file is multiple files in a folder. I can't seem to get the manager to work with the game. Game in question is "Deaths Gambit : Afterlife".

**Comment by metehankutlu** [comment]
Yes the plugin won't accept multiple files for now but I have plans to add a feature for multiple files in the next version when I have time.
